### ISSUE TITLE
File busy on platform failure

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1271]
In GitLab by @dbeltrankyl on Apr 9, 2024, 10:08

Follow up on an issue raised in the production experiment

```
[ERROR] 2024-04-08 18:26:15,008 TRACE:
Can not send file /appl/AS/AUTOSUBMIT_DATA/a0py/tmp/ASLOGS/submit_LUMI.sh to /scratch/project_465000454/lrb_465000454_asub/a0py/LOG_a0py/ASLOGS/submit_LUMI.sh JOBS:['a0py_20200101_fc0_460_13_OPA_URBAN_1']
```

Autosubmit, disconnected from the platform, and reconnected again, until here, all fine, but subsequent submissions complained about `text file busy`

`Command /scratch/project_465000454/lrb_465000454_asub/a0py/LOG_a0py/submit_LUMI.sh in lumi-cluster warning: bash: /scratch/project_465000454/lrb_465000454_asub/a0py/LOG_a0py/submit_LUMI.sh: Text file busy`

`Command /scratch/project_465000454/lrb_465000454_asub/a16z/LOG_a16z/submit_LUMI.sh in lumi-cluster warning: bash: /scratch/project_465000454/lrb_465000454_asub/a16z/LOG_a16z/submit_LUMI.sh: Text file busy
`

So it seems the first error that blocked the file for unknown reasons (could be a bug in the paramiko library that was updated not too much ago), making it unusable until autosubmit was stopped.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Apr 9, 2024, 10:16

What I plan to do instead of reusing the same file name is to add a .tmp random name FYI @kinow

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on May 10, 2024, 10:38

mentioned in issue digital-twins/de_340-2/project_management#336

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jun 11, 2024, 11:32

Implemented in 4.1.6, now autosubmit creates a tmp file + deletes it each time
